frontend.sh

```diff
@@ -21,6 +21,10 @@ init() {
     DEP_CHECKSUM=$(cat package.json | shasum -a 256)
   fi
 
+  if [[ "$(node -v)" != 'v8.'* ]]; then
+    printf "\033[1;31mPlease install Node 8.x\033[0m\n"
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3177/commits/8a0151ec8aefacc82ff99913c70a6b824c78fd36